Sources/hostmgr-helper/VirtualMachineSlot.swift

```diff
@@ -63,52 +62,53 @@ class VirtualMachineSlot: NSObject, ObservableObject, VZVirtualMachineDelegate {
         } catch {
             Logger.helper.error("Error launching VM: \(error.localizedDescription)")
             Logger.helper.error("Attempting Cleanup of \(launchConfiguration.handle)")
-            try await vmManager.removeVM(name: launchConfiguration.handle)
 
+            try? await VMManager.removeVM(name: launchConfiguration.handle)
```

Comment:
Worth wrapping the call inside a `do { try await … } catch { Logger.helper.error("Failed to remove VM file: \(error)") }` to at least log the details of the error occurred during removal attempt, even if we don't want it to throw?

Reply:
Handled in 4c58da176db8a8cc297bb1818967e9ff2f611902.